### ISSUE TITLE
How to search for particular things stored this can be anything that is stored I would like either time or something and is there a way to earase all data of a particular user

### ISSUE THREAD

**Issue by furkanshaikh313** [jina-ai/vectordb#56]

**Comment by JoanFM** [comment]
Hello @furkanshaikh313,

Can you be more explicit? What are you trying to search? This library is to search based on embeddings

**Comment by furkanshaikh313** [comment]
I want to be able to save different types of data inside a vector database with having the ability to retrive data easily @JoanFM

**Comment by JoanFM** [comment]
What kind of data you want to store. Right now, vectordb allows to store data in a specific format schema

**Comment by furkanshaikh313** [comment]
Like mainly text but it would be amazing if i can save images and audio as well

**Comment by JoanFM** [comment]
You can define the schema like this:

```
from docarray import BaseDoc
from docarray.documents import ImageDoc, TextDoc, AudioDoc

class MyDoc(BaseDoc):
    text: Optional[TextDoc]
    image: Optional[ImageDoc]
    audio: Optional[AudioDoc]
    embedding: NdArray
```

**Comment by furkanshaikh313** [comment]
so should i use this with vector db?

**Comment by JoanFM** [comment]
yes, u can use this, define a schema that satisfies ur needs, and u can do this

**Comment by furkanshaikh313** [comment]
by the way @JoanFM do you know a way i can like host this 24/7 on cloud for free?

**Comment by JoanFM** [comment]
I do not think there are options to host for free.

**Comment by furkanshaikh313** [comment]
hmm Listen I am thinking of using this on a big project of mine and it need to be able to handle different customers and not intermix anything is that possible?

**Comment by JoanFM** [comment]
multitenancy is not supported. You would need to create an index for each of ur customers
